GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/StillThereViewModel.cs

```diff
@@ -112,7 +111,7 @@ private void OneSecondElapsed(object sender, System.EventArgs e)
             CurrentSeconds--;
             decimal StartingSeconds = 30;
             decimal PercentOfSeconds = (CurrentSeconds / StartingSeconds) * 100;
-            Percentage = Convert.ToInt16(Math.Floor(PercentOfSeconds));
+            Percentage = System.Convert.ToInt16(System.Math.Floor(PercentOfSeconds));
```

Comment:
Why did you remove the `System` import?

Reply:
Yeah, sorry don't mean to be flip flopping. Can you import `System.Convert`?

Reply:
Nah. That's unavailable as a sole import. There are also other items on this view model using `System` eg. `System.Action`.

Reply:
I wound up importing `system` since it was used throughout this component, but I'm open to changing this is needed.